lizmap/www/assets/css/admin.css

```
@@ -101,6 +100,18 @@ fieldset {
     }
 }
 
+/*
+======
+Admin Repositry lists
+======
+*/
+div.admin_repository:target::before {
+    /* Fixed page header overlaps in-page anchors https://stackoverflow.com/questions/4086107/fixed-page-header-overlaps-in-page-anchors */
```

```suggestion
    /* Fixed page header overlaps in-page anchors 
     * https://stackoverflow.com/questions/4086107/fixed-page-header-overlaps-in-page-anchors */
```

Otherwise, this line is too long